pkg/agent/openflow/pipeline.go

```diff
@@ -438,411 +450,543 @@ func (c *client) DeleteOFEntries(ofEntries []binding.OFEntry) error {
-						MatchCTStateTrk(true).
-						Action().CT(false, ServiceConntrackCommitTable.GetNext(), snatZone).
-						NAT().
-						CTDone().
-						Done(),
-				)
-			}
-		}
-	} else {
-		flows = append(flows, c.kubeProxyFlows(category)...)
+// Feature: PodConnectivity
+// Stage: ClassifierStage
+// Tables: ClassifierTable
+// Refactored from:
+//   - func (c *client) podUplinkClassifierFlows(dstMAC net.HardwareAddr, category cookie.Category) (flows []binding.Flow)
+// podUplinkClassifierFlows generates the flows to mark the packets with target destination MAC address from uplink/bridge
+// port, which are needed when uplink is connected to OVS bridge and Antrea IPAM is configured.
+func (f *featurePodConnectivity) podUplinkClassifierFlows(dstMAC net.HardwareAddr) []binding.Flow {
+	return []binding.Flow{
+		// This generates the flow to mark the packets from uplink port.
+		ClassifierTable.ofTable.BuildFlow(priorityHigh).
+			Cookie(f.cookieAllocator.Request(f.category).Raw()).
+			MatchInPort(config.UplinkOFPort).
+			MatchDstMAC(dstMAC).
+			Action().LoadRegMark(FromUplinkRegMark).
+			Action().GotoStage(binding.ConntrackStateStage).
+			Done(),
+		// This generates the flow to mark the packets from bridge local port.
+		ClassifierTable.ofTable.BuildFlow(priorityHigh).
+			Cookie(f.cookieAllocator.Request(f.category).Raw()).
+			MatchInPort(config.BridgeOFPort).
+			MatchDstMAC(dstMAC).
+			Action().LoadRegMark(FromBridgeRegMark).
+			Action().GotoStage(binding.ConntrackStateStage).
+			Done(),
 	}
+}
 
-	// TODO: following flows should move to function "kubeProxyFlows". Since another PR(#1198) is trying
-	//  to polish the relevant logic, code refactoring is needed after that PR is merged.
-	for _, proto := range c.ipProtocols {
-		ctZone := CtZone
-		if proto == binding.ProtocolIPv6 {
-			ctZone = CtZoneV6
-		}
+// Feature: PodConnectivity
+// Stage: ConntrackStateStage
+// Tables: ConntrackTable, ConntrackStateTable
+// Stage: ConntrackStage
+// Tables: ConntrackCommitTable
+// Refactored from:
+//   - func (c *client) connectionTrackFlows(category cookie.Category) []binding.Flow
+//   - func (c *client) conntrackBasicFlows(category cookie.Category) []binding.Flow
+//   - part of func (c *client) serviceLBBypassFlows(ipProtocol binding.Protocol) []binding.Flow
+// Modifications:
+//   - Remove the flows related with Service since they are for feature Service.
+// conntrackFlows generates the flows about conntrack for feature PodConnectivity.
+func (f *featurePodConnectivity) conntrackFlows() []binding.Flow {
+	var flows []binding.Flow
+	for _, ipProtocol := range f.ipProtocols {
 		flows = append(flows,
-			// Connections initiated through the gateway are marked with FromGatewayCTMark.
-			ConntrackCommitTable.BuildFlow(priorityNormal).MatchProtocol(proto).
+			// This generates the flow to maintain tracked connection in CT zone.
+			ConntrackTable.ofTable.BuildFlow(priorityNormal).
+				Cookie(f.cookieAllocator.Request(f.category).Raw()).
```

Comment:
you can also define a `cookieID` as `f.cookieAllocator.Request(f.category).Raw()` here. 
you probably need to check a few other functions.

Reply:
Done.